doc/JSON_INFO.md

```diff
@@ -1278,6 +1278,33 @@ See also VEHICLE_JSON.md
 "symbol": "[",                   // The item symbol as it appears on the map. Must be a Unicode string exactly 1 console cell width.
 "looks_like": "rag",              // hint to tilesets if this item has no tile, use the looks_like tile
 "description": "Socks. Put 'em on your feet.", // Description of the item
+"ascii_picture": [
+      "        ,,,,,,,,,,,,,",
+      "    .;;;;;;;;;;;;;;;;;;;,.",
+      " .;;;;;;;;;;;;;;;;;;;;;;;;,",
+      ".;;;;;;;;;;;;;;;;;;;;;;;;;;;;.",
+      ";;;;;@;;;;;;;;;;;;;;;;;;;;;;;;' .............",
+      ";;;;@@;;;;;;;;;;;;;;;;;;;;;;;;'.................",
+      ";;;;@@;;;;;;;;;;;;;;;;;;;;;;;;'...................`",
+      ";;;;@;;;;;;;;;;;;;;;@;;;;;;;'.....................",
+      " `;;;;;;;;;;;;;;;;;;;@@;;;;;'..................;....", // Ascii art that  will be displayed at the bottom of the item description. Array of string with each element being a line of the picture.
+      "   `;;;;;;;;;;;;;;;;@@;;;;'....................;;...",
+      "     `;;;;;;;;;;;;;@;;;;'...;.................;;....",
+      "        `;;;;;;;;;;;;'   ...;;...............;.....",
+      "           `;;;;;;'        ...;;..................",
+      "              ;;              ..;...............",
+      "              `                  ............",
+      "             `                      ......",
+      "             `                         ..",
+      "           `                           '",
+      "          `                           '",
+      "         `                           '",
+      "        `                           `",
+      "        `                           `,",
+      "        `",
+      "         `",
+      "           `."
+    ],
```

Comment:
While an example is good, I definitely think we want to mention what the typical line widths are in the common interfaces so that artists don't have to guess at whether or not their design will fit.